### PR TITLE
Added examples to du.md

### PR DESCRIPTION
**Pull request by ireaneus**
- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/du.md [hunk 1]
```diff
@@ -14,14 +14,30 @@
...
 
 `du -h --max-depth=N {{path/to/directory}}`
 
 - List the human-readable size of all .jpg files in subdirectories of the current directory, and show a cumulative total at the end:
 
 `du -ch */*.jpg`
+
+- Number of files search:
+
+``echo "Detailed Inode usage for: $(pwd)" ; for d in `find -maxdepth 1 -type d |cut -d\/ -f2 |grep -xv . |sort`; do c=$(find $d |wc -l) ; printf "$c\t\t- $d\n" ; done ; printf "Total: \t\t$(find $(pwd) | wc -l)\n"``
```
Comment (zdroid) [inline pages/common/du.md]: It is only one command per example. So if you want to add 3 commands, you need text for all 3. That's causing the error in Travis, too.

Comment (ireaneus) [reply pages/common/du.md]: Ok, Yea I have to encapsulate `find -maxdepth 1 -type d |cut -d\/ -f2 |grep -xv . |sort` this in an executable bash string that is not using the back tic too.  Also I thought that we could have up to 8 examples per command?

**Comment by zdroid** [comment]
Have you tried GitHub's Gist, @sbrl?

@ireaneus I have to say I liked the pun in the last commit message. :dancer:

**Comment by sbrl** [comment]
I do actually use GitHub gists at the moment with _Lepton_ to manage my snippets, but it's far from ideal. The UI is not configurable at all, and I don't really like it. Moreover, I want to keep it locally / self-hosted.

**Comment by ireaneus** [comment]
@agnivade, yes I see your point and now understand what tldr format and meaning is used for.  I will close this pull request.

**Pull request closed without merging**